.gitlab-ci-check-docker-acceptance.yml

```diff
@@ -117,3 +117,41 @@ publish:acceptance:
         fi;
         sleep 1;
       done
+
+publish:acceptance:
+  stage: publish
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  image: golang:1.14-alpine3.11
+  dependencies:
+    - test:acceptance_tests
+  before_script:
+    - apk add --no-cache git
+    # Run go get out of the repo to not modify go.mod
+    - cd / && go get github.com/mattn/goveralls && cd -
+    # Coveralls env variables:
+    #  According to https://docs.coveralls.io/supported-ci-services
+    #  we should set CI_NAME, CI_BUILD_NUMBER, etc. But according
+    #  to goveralls source code (https://github.com/mattn/goveralls)
+    #  many of these are not supported. Set only CI_PR_NUMBER and
+    #  pass few others are command line arguments.
+    #  See also https://docs.coveralls.io/api-reference
+    - export CI_PR_NUMBER=${CI_COMMIT_BRANCH#pr_}
+  script:
+    - goveralls
+      -repotoken ${COVERALLS_TOKEN}
+      -service gitlab-ci
+      -jobid $CI_PIPELINE_ID
+      -covermode set
+      -flagname acceptance
+      -parallel
+      -coverprofile ./tests/coverage-acceptance.txt
+
+coveralls:finish-build:
+  stage: .post
+  # See https://docs.coveralls.io/parallel-build-webhook
+  variables:
+    COVERALLS_WEBHOOK_URL: "https://coveralls.io/webhook"
+  image: appropriate/curl
+  script:
+    - 'curl -k ${COVERALLS_WEBHOOK_URL}?repo_token=${COVERALLS_TOKEN} -d "payload[build_num]=$CI_PIPELINE_ID&payload[status]=done"'
```

Comment:
Question:
I thought this could be done with `goveralls` also, but I guess curl was just as easy?

Reply:
It is just faster: CI uses this tiny Docker image with curl, instead of a golang enabled image + downloading `goveralls`.